llvm/test/CodeGen/AIE/aie2/schedule/interblock/memdeps.mir

```diff
@@ -34,6 +34,7 @@ body:             |
   ; CHECK-NEXT:   VST_SRS_D8_S32_ag_idx_imm killed $p0, 0, killed $cm0, killed $s0, implicit-def $srsrs_of, implicit $crsat, implicit $crrnd, implicit $crsrssign
   ; CHECK-NEXT:   NOP
   ; CHECK-NEXT:   NOP
+  ; CHECK-NEXT:   NOP
```

Comment:
Why do you think it will be a big change if we need to infer `none` to `DM` ?

Note : access to tile memory is always annotated at intrinsic level so we will always infer that

Reply:
I mean it will be an invasive change, because tests that go from IR to MIR will get different MMOs if the default becomes `DM` instead of `none`